tests/infra/remote_client.py

```diff
@@ -39,8 +41,13 @@ def __init__(
         self.BIN = infra.path.build_bin_path(bin_path)
 
         # strip out the config from the path
+        self.common_dir = infra.ccf.get_common_folder_name(workspace, label)
 
-        self.DEPS = glob("*.pem") + [config]
+        self.DEPS = [
+            os.path.join(self.common_dir, "user1_cert.pem"),
```

Comment:
why is user1 in DEPS?

Reply:
This is required by the `small_bank_client` to authenticate itself when talking to CCF. This is not ideal but it was there before (in fact, we were copying _all_ the PEM files to the client directory) so I think it can be addressed in a different PR.